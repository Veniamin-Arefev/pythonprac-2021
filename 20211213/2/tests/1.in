(rand := __import__("random"), rand.seed("Taki testik"), sorted([rand.randint(-100, 100) for i in range(16)], reverse=True))[2]