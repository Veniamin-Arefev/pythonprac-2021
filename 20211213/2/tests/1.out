[-82, -46, -40, -24, -13, -12, -12, -6, 16, 16, 50, 50, 67, 78, 90, 92]
